bin/processGsRawData.sh

```diff
@@ -463,23 +464,21 @@ function processSamplesheetsAndMoveCovertedData() {
 		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "${_controlFileBaseForFunction}.finished not present -> Continue..."
 		printf '' > "${_controlFileBaseForFunction}.started"
 	fi
-	
 	#
 	# Combine GenomeScan samplesheet per batch with inhouse samplesheet(s) per project.
 	#
 	createInhouseSamplesheetFromGS.py \
-		--GenomeScanInputDir "${TMP_ROOT_DIR}/${_batch}/" \
-		--logfile "${_logFile}" \
-		--samplesheetNewDir "${TMP_ROOT_DIR}/Samplesheets/new/" \
-		--samplesheetOutputDir "${TMP_ROOT_DIR}/${_batch}/" \
+		--genomeScanInputDir "${TMP_ROOT_DIR}/${_batch}/" \
+		--inhouseSamplesheetsInputDir "${TMP_ROOT_DIR}/Samplesheets/new/" \
+		--samplesheetsOutputDir "${TMP_ROOT_DIR}/${_batch}/" \
+		--logFile "${_logFile}" \
 		>> "${_logFile}" 2>&1 \
```

Comment:
how does this work? You are having logFile as output file and also redirecting to that same logFile?

Reply:
Good point. This was unchanged as compared to the previous version (only indentation of changed), and it did not cause problems previously, but it's a bit silly. I suggest to remove the --logFile option for processGsRawData.sh and simply write any status messages to STDOUT. If you want to log, you can always redirect to a log file. Is that Ok?

Reply:
yep ,sounds good to me

Reply:
reminder :)

Reply:
Check, updated in last commit...